resources/web/style/toc.pcss

```diff
@@ -0,0 +1,86 @@
+#guide {
+  .toc {
+    ul {
+      margin: 0 0 1em 0;
+      padding: 0;
+      border: 1px solid #ddd;
+      ul {
+        /* Hide all sub-toc elements by default. See rules with `.show` that
+         * show specific sub-toc elements. */
+        display: none;
+      }
+    }
+    li {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+      ul {
+        border: none;
+      }
+    }
+    span {
+      display: block;
+      padding: 0.5em 0;
+      font-size: 0.85em;
+      line-height: 1.3em;
+    }
+
+    .current_page {
+      font-weight: bold;
+    }
+
+    .collapsible.show {
+      > ul {
+        display: block;
+      }
+      > span {
+        background-image: inline("img/minus.png");
+        background-color: white;
+        border-bottom: 1px solid white;
+      }
+    }
+    .collapsible {
+      > span {
+        background-repeat: no-repeat;
+        background-image: inline("img/plus.png");
+        &:hover {
+          background-color: #fafafa;
+          cursor: pointer;
+        }
+      }
+    }
+
+    /* Customize each level of the TOC, mostly so it looks "indented". */
+    > li {
+      > span {
+        background-color: #efefef;
+        border-bottom: 1px solid #ddd;
+        font-size: 1em;
+        background-position: 0 11px;
```

Comment:
These background positions aren't the same as they used to be. I didn't like how they lined up before and they were defined all over the place. I've grouped them here which makes me feel much better about them.